syntax = "proto2";

package Proto;


message TargetScenario {
    message Launch {
        required double time = 1;
        required double angle_pos = 2; // degrees

        // all random if not set
        optional double abs_speed = 3;
        optional double angle_deviation = 4; // degrees
        optional double height = 5;
        optional bool is_accurate = 6;
    }

    repeated Launch launches = 1;
}
